### ISSUE TITLE
explanation of seed in agent 

### ISSUE THREAD

**Issue by twyla19** [mdavolio/mancala#17]
can you explain what is the purpose of seed in every agent?

**Comment by user01** [comment]
Agents have stochastic components, but all operations should be deterministic for reproducibility, debugging, and testing.
The seed fixes the [PRNG](https://en.wikipedia.org/wiki/Pseudorandom_number_generator) so based on the seed, the random will exhibit the same close to truly random behavior.

Some explicitly, such as the [Random Agent](https://github.com/mdavolio/mancala/blob/56008a1f575f3b44b34ca85e79ab243f67b6eade/mancala/agents/random.py#L27)
Others for edge cases, such as the [Exact Agent](https://github.com/mdavolio/mancala/blob/56008a1f575f3b44b34ca85e79ab243f67b6eade/mancala/agents/exact.py#L65)

In short, seeds let agents behave randomly from an experimental/exploration angle and consistently from an operational/production angle.

**Comment by twyla19** [comment]
Is there any specific/scientific reason why it's 451?
And if I create a new agent with a different algorithm, should I write the seed 451 as well so it can balance each other?

I was thinking what if I put MCTS agent on it.
